src/js/profile/chart.js

```diff
@@ -196,6 +196,9 @@ export class Chart extends Observable {
               height: { scale: "yscale", band: 1 },
               x: { scale: "xscale", field: { signal: "datatype[Units]" } },
               x2: { scale: "xscale", value: 0 },
+              tooltip: {
+                signal: "{'percentage': format(datum.percentage, numberFormat.percentage), 'group': mainGroup, 'count': format(datum.count, numberFormat.value)}"
```

Comment:
sorry my mistake `group` should be  `group: datum[mainGroup]`

Reply:
Refactored